modules/ensemblvep/main.nf

```diff
@@ -1,12 +1,13 @@
 process ENSEMBLVEP {
+    tag "$meta.id"
     label 'process_medium'
 
     conda (params.enable_conda ? "bioconda::ensembl-vep=104.3" : null)
-    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        !task.ext.container_tag ?
+    container "${ task.ext.container_tag ?
+            "nfcore/vep:${task.ext.container_tag}" :
```

Comment:
These quotes here should be single quotes `'`. They're closing the first string and opening a new one after the curly brace.

Reply:
Oh, no, then the variable interpolation doesn't work. 
I think the string should be:
```
"${ task.ext.container_tag ? 'nfcore/vep:'+task.ext.container_tag : workflow....
```

Reply:
Are you variable interpolation is not working? CI is working

Reply:
Sorry, I should not be looking at code when I'm super tired. For some reason I thought there was an issue when there wasn't. Seems to keep happening on your pull requests too.

Reply:
I'm doing weird stuff too